.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+name: rospkg-ci
+
+on:
+  push:
+    branches: ['master']
```

Comment:
You didn't use quotes in the other PR. It would be nice to be consistent - I don't care which way.

Reply:
Hah. What comes of making edits while being social at ROS World. Dropped quotes.
```suggestion
    branches: [master]
```

Reply:
Thanks for spotting.